Formula/llvm.rb

```diff
@@ -444,6 +444,21 @@ def caveats
   end
 
   test do
+    # Use brewed glibc if installed.
+    glibc_args = if OS.linux?
+      glibc = Formula["glibc"]
+      glibc_installed = glibc.any_version_installed?
+      if glibc_installed
+        [
+          "-nostdlib",
+          "-Wl,-dynamic-linker=#{HOMEBREW_PREFIX}/lib/ld.so",
+          "#{Formula["gcc"].opt_lib}/gcc/current/gcc/x86_64-pc-linux-gnu/12/Scrt1.o",
```

Comment:
Can we use `#{Formula["glibc"].opt_lib}/Scrt1.o` instead here?  That works for me locally.  Also I do not understand why we have to manually supply this - `clang` should automatically be adding this.

Reply:
```console
$ readlink ~/.linuxbrew/opt/gcc/lib/gcc/current/gcc/x86_64-pc-linux-gnu/12/Scrt1.o
/home/linuxbrew/.linuxbrew/opt/glibc/lib/Scrt1.o
```
Yes! Thanks for the suggestion. I didn't notice that GCC's `Scrt1.o` is a symlink to Glibc.